app/workers/notifications/milestone_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Notifications
-  class MilestoneJob < ApplicationJob
-    queue_as :send_milestone_notification
+  class MilestoneWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
 
     def perform(type, article_id, service = Notifications::Milestone::Send)
```

Comment:
Since we are not using the service argument here I think we should remove it in favor of explicitly defining the service in the worker.

Reply:
Something like:
```ruby
Notifications::Milestone::Send.call(type, article)
```
?

Reply:
Exactly

Reply:
Ok, I will send the changes

Reply:
@mstruve I sent the changes without updating the specs. I'm doing it right now